.github/lineage.yml

```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-ansible-role.git
```

Comment:
Is this the correct upstream?

Reply:
Yes, this repo is the Ansible role to install the [PCA GoPhish Docker composition](https://github.com/cisagov/pca-gophish-composition).

Reply:
I am a fool.  I misread the repo name.  Apologies for my buffoonhood.

Reply:
The TravisCI builds fail.  Do you care?

Reply:
I don't care about the Travis failures- it will be removed soon enough.
I realize I can pull in the upstream changes on my own, but I am letting Lineage do its thing.

Reply:
thing --> thang

OK.